src/test/resources/com/microsoft/jenkins/azuread/integrations/casc/ConfigAsCodeTest/config.xml

```diff
@@ -0,0 +1,58 @@
+<hudson>
+    <disabledAdministrativeMonitors/>
+    <version>2.138.3</version>
+    <installStateName>DEVELOPMENT</installStateName>
+    <numExecutors>2</numExecutors>
+    <mode>NORMAL</mode>
+    <useSecurity>true</useSecurity>
+    <authorizationStrategy class="com.microsoft.jenkins.azuread.AzureAdMatrixAuthorizationStrategy">
+        <permission>com.cloudbees.plugins.credentials.CredentialsProvider.Create:authenticated</permission>
+        <permission>com.cloudbees.plugins.credentials.CredentialsProvider.ManageDomains:authenticated</permission>
+        <permission>com.cloudbees.plugins.credentials.CredentialsProvider.View:authenticated</permission>
+        <permission>hudson.model.Computer.Build:authenticated</permission>
+        <permission>hudson.model.Computer.Connect:authenticated</permission>
+        <permission>hudson.model.Computer.Create:authenticated</permission>
+        <permission>hudson.model.Computer.Delete:authenticated</permission>
+        <permission>hudson.model.Computer.Disconnect:authenticated</permission>
+        <permission>hudson.model.Hudson.Administer:anonymous</permission>
+        <permission>hudson.model.Hudson.Administer:authenticated</permission>
+        <permission>hudson.model.Item.Build:authenticated</permission>
+        <permission>hudson.model.Item.Cancel:authenticated</permission>
+        <permission>hudson.model.Item.Configure:authenticated</permission>
+        <permission>hudson.model.Item.Read:authenticated</permission>
+        <permission>hudson.model.Run.Delete:authenticated</permission>
+        <permission>hudson.model.Run.Update:authenticated</permission>
+        <permission>hudson.model.View.Configure:authenticated</permission>
+        <permission>hudson.model.View.Delete:abc (abc@jenkins.com)</permission>
```

Comment:
I was comparing this with the ```config.xml``` generated in my Jenkins instance. Is this equivalent to 
```<permission>com.cloudbees.plugins.credentials.CredentialsProvider.Create:AZURE_AD_GROUP_NAME_HERE (37e4c133-6c0e-458f-9ca7-c15fda3e5b60)</permission>``` ?

Reply:
Yes, UPN and object id are all supported in this plugin for now.